src/test/common_helper.py

```diff
@@ -320,6 +320,15 @@ def get_file_name(self, uid):
         if uid == 'deadbeef00000000000000000000000000000000000000000000000000000000_123':
             return 'test_name'
 
+    def set_unpacking_lock(self, uid):
+        pass
+
+    def check_unpacking_lock(self, uid):
```

Comment:
check_unpacking_lock and release_unpacking_lock are currently unused by the tests